.chloggen/ibraheem_enable_instrumentation_scope_metadata_as_tags_by_default_in_datadogexporter.yaml

```diff
@@ -0,0 +1,13 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable instrumentation_scope_metadata_as_tags by default in datadogexporter. Scope attributes are now added as tags to metrics.
```

Comment:
you can add a note in the subtext that this may lead to metric tag cardinality issue if users have too many unique instrumentation scopes. If so they will want to disable instrumentation_scope_metadata_as_tags